crypto/provider_core.c

```diff
@@ -342,7 +342,7 @@ static const OSSL_DISPATCH *core_dispatch; /* Define further down */
  * locking.  Direct callers must remember to set the store flags when
  * appropriate
  */
-static int provider_activate(OSSL_PROVIDER *prov)
+static int provider_activate(OSSL_PROVIDER *prov, OPENSSL_CTX *libctx)
```

Comment:
I see this as a temporary hack, 'cause this *assumes* the provider uses a library context as a provider context...  which happens to be true for the *current* FIPS module implementation...  so we need to make sure there are notes, both here and in `fipsprov.c` that this parameter must be what the module uses, or else...

(Here be dragons)

Reply:
Extra commentary that goes into this noted, and appreciated